src/components/com_tjcertificate/administrator/libraries/mails.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * @package     TJCertificate
+ * @subpackage  com_tjcertificate
+ *
+ * @author      Techjoomla <extensions@techjoomla.com>
+ * @copyright   Copyright (C) 2009 - 2020 Techjoomla. All rights reserved.
+ * @license     http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL
+ */
+
+defined('_JEXEC') or die;
+jimport('techjoomla.tjnotifications.tjnotifications');
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\User\User;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Component\ComponentHelper;
+use Joomla\Registry\Registry;
+
+/**
+ * Class TjCertificateMails
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class TjCertificateMails
+{
+	protected $params;
+
+	protected $siteConfig;
+
+	protected $sitename;
+
+	protected $siteadminname;
+
+	protected $user;
+
+	protected $client;
+
+	protected $tjnotifications;
+
+	protected $siteinfo;
+
+	/**
+	 * Method acts as a consturctor
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function __construct()
+	{
+		$this->params              = ComponentHelper::getParams('com_tjcertificate');
+		$this->siteConfig          = Factory::getConfig();
+		$this->sitename            = $this->siteConfig->get('sitename');
+		$this->siteadminname       = $this->siteConfig->get('fromname');
+		$this->user                = Factory::getUser();
+		$this->client              = "com_tjcertificate";
+		$this->tjnotifications     = new Tjnotifications;
+
+		$this->siteinfo            = new stdClass;
+		$this->siteinfo->sitename  = $this->sitename;
+	}
+
+	/**
+	 * Send mails when record is created
+	 *
+	 * @param   OBJECT  $recordDetails  Campaigns Detail
```

Comment:
Update function comment...check in every function of all the files

Reply:
done